app/js/controllers.js

```diff
@@ -4433,6 +4434,9 @@ angular.module('myApp.controllers', ['myApp.i18n'])
 
       $scope.notify.preview = !settings[4]
 
+      // Set default to false
+      $scope.theme.darkmode = settings[6];
```

Comment:
Again, make this more generic:

```
$scope.theme = settings[6]
```

This should not be a boolean, but rather a string value, e.g., `settings[6] = 'dark'`.